br/pkg/gluetidb/glue.go

```diff
@@ -226,11 +226,11 @@ func (gs *tidbSession) SplitBatchCreateTable(schema model.CIStr, info []*model.T
 			return err
 		}
 		mid := len(info) / 2
-		err = gs.SplitBatchCreateTable(schema, info[:mid])
+		err = gs.SplitBatchCreateTable(schema, info[:mid], cs...)
```

Comment:
Maybe do not use optional arguments here?

Reply:
remote opt parameter in https://github.com/pingcap/tidb/pull/41358/commits/dd7388a722999c8999664332fbdf694ff03b866a